js/tween/tweener.babel.js

```diff
@@ -134,11 +134,13 @@ class Tweener {
     @private
   */
   _restorePlayingTweens () {
-    for (let i = 0; i < this._savedTweens.length; i++ ) {
-      this._savedTweens[i].resume();
+    if (this._savedTweens !== undefined && this._savedTweens.constructor === Array && this._savedTweens.length) {
```

Comment:
let's just check if `this._savedTweens` is defined, all subsequent check are redundant, something like:
```javascript
if (this._savedTweens == null) { return; } // this will cover both cases `undefined` and `null`
// loop
```